src/components/cart/EmptyCart/EmptyCart.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import Button from 'src/components/ui/Button'
+import { ShoppingCart as ShoppingCartIcon } from 'phosphor-react'
+import { useCartToggleButton } from 'src/sdk/cart/useCartToggleButton'
+
```

Comment:
```suggestion

import './empty-cart.scss '

```

Reply:
Oh, thanks for seeing that! 
I totally forgot about it.